### PR TITLE
USSD Public - lookup service rating status instead of contact

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Get the status from the service rating endpoint. Example: https://github.com/praekelt/seed-service-rating/blob/feature/issue-1-initial-port/ratings/tests.py#L164-L172

The results looks like: 

```
{
    "count": 1,
    "next": null,
    "previous": null,
    "details": [{
        "updated_at": "2016-04-04T17:06:08.411867Z",
        "created_at": "2016-04-04T17:06:08.411843Z",
        "version": 1,
        "expired": false,
        "id": "1b47bab8-1c37-44a2-94e6-85c3ee9a8c8b",
        "identity": "210ac8c7-1f23-46af-a186-2468c89f7cc1",
        "completed": false,
        "invited": false,
        "expires_at": null,
        "invite": {
            "content": "Please dial *120*1234# and rate our service",
            "to_addr": "+27123",
            "metadata": {}
        }
    }]
}
```

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps should we direct users to the service rating flow only if invited has been set to true first..?  When (invited && !completed && !expired) evaluates to true..?

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas sorry that link is broken now, and probably incomplete. Should be something like:
`/api/v1/invite/?identity=usersUUID&completed=False&expired=False`

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review.

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps can't get fixtures to work for '/api/v1/invite/identity=usersUUID&completed=False&expired=False'... please help.

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: src/ussd_public.js [hunk 1]
```diff
@@ -744,13 +769,20 @@ go.app = function() {
...
+                }
             });
         });
 
-        // EndState
+        // EndState 6
         self.add('state_end_servicerating', function(name) {
-            // set servicerating_unanswered flag to false...
+            // sets servicerating_unanswered to false indicating completed servicerating feedback
+            self.im.user.set_answer('servicerating_unanswered', false);
```
Comment (imsickofmaps) [inline src/ussd_public.js]: This is not needed now, but we do need to do a PATCH to the `invite` to set `completed` to `true`.

**Comment by imsickofmaps** [comment]
Looking better 😄 ! Can you just do the util function that PATCH's the invite then I'll do a quick review again

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review the latest changes you requested

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**